### PR TITLE
Fix getting bounds for non existing elements preventing BROWSER_TIMEOUT

### PR DESCRIPTION
**Pull request by z00sts**

### CODE REVIEW

**Comment by imurchie** [comment]
Thanks for this!

There was a problem with the Travis configuration, so the build failed. I've fixed it in the master branch. Could you rebase and re-push here?

File: bootstrap/src/io/appium/android/bootstrap/AndroidElementsHash.java [hunk 1]
```diff
@@ -99,9 +99,16 @@ public AndroidElement getElement(final UiSelector sel, final String key)
...
+      } catch (final UiObjectNotFoundException e) {
+        throw new ElementNotFoundException();
+      }
+    }
+
+    if (el.exists()) {
+      // there are times when UiAutomator returns an element from another parent
+      // so we need to see if it is within the bounds of the parent
+      try {
+        if (baseEl != null && !Rect.intersects(baseEl.getBounds(), el.getBounds())) {
```
Comment (z00sts) [inline bootstrap/src/io/appium/android/bootstrap/AndroidElementsHash.java]: Added `baseEl != null` for simple search case (with no parent)

Comment (imurchie) [inline bootstrap/src/io/appium/android/bootstrap/AndroidElementsHash.java]: Nice! Good catch.

**Comment by imurchie** [comment]
Published in `appium-android-bootstrap@2.7.8`.

**Pull request merged**